packages/webapp/src/utils/acounts.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Utilities to deal with burner accounts.
+ *
+ * @module utils/accounts
+ */
+
+import {
+    createPublicClient,
+    createWalletClient,
+    http,
+    getContract,
+    concat,
+    encodeFunctionData,
+} from 'viem'
+import { localhost } from 'viem/chains'
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts'
+import {
+    simpleAccountFactoryAbi,
+    simpleAccountAbi,
+    paymasterAbi
+} from 'src/utils/abis'
+
+interface UserOperation {
+    sender: `0x${string}`
+    nonce: number
+    initCode: `0x${string}`
+    callData: `0x${string}`
+    callGasLimit: number
+    verificationGasLimit: number
+    preVerificationGas: number
+    maxFeePerGas: number
+    maxPriorityFeePerGas: number
+    paymasterAndData: `0x${string}`
+    signature: `0x${string}`
+}
+
+const publicClient = createPublicClient({
+    chain: localhost,
+    transport: http()
+})
+
+const paymasterAddress = process.env.PAYMASTER_ADDRESS as `0x${string}`
+const paymaster = getContract({
+    address: paymasterAddress,
+    abi: paymasterAbi,
+    publicClient
+})
+
+export async function createAccount() {
+
+    // use simple account factory to deploy a new smart account
+    // get the address of the smart account
+    // create sdk that reads private key from the browser storage and signs transactions
+
+    const privateKey = generatePrivateKey()
+    // store this private key in browser storage (is this secure?)
+    localStorage.setItem("privateKey", privateKey)
+
+    const account = privateKeyToAccount(privateKey)
+
+    // TODO: change chain
+    const walletClient = createWalletClient({
+        account,
+        chain: localhost,
+        transport: http()
+    })
+
+    // Generate initcode
+    const simpleAccountFactoryAddress = process.env.SIMPLE_ACCOUNT_FACTORY_ADDRESS as `0x${string}`
+    const simpleAccountFactory = getContract({
+        address: simpleAccountFactoryAddress,
+        abi: simpleAccountFactoryAbi,
+        publicClient
+    })
+    const initCode = concat([
+        simpleAccountFactoryAddress,
+        encodeFunctionData({
+            abi: simpleAccountFactoryAbi,
+            functionName: 'createAccount',
+            args: [account.address, 0]
+        }) // set salt as 0
+    ])
+
+    // Generate calldata to execute a transaction
+    const simpleAccountAddress = await simpleAccountFactory.read.getAddress([account.address, 0]) as `0x${string}`
```

Comment:
Using wagmi CLI (cf. prev comment) you can get a function in `generated.ts` that does this call without the need to piece to do the `getContract` call yourself.

(And same for all other usages of `getContract`, of course.)